Cargo.toml

```
@@ -25,6 +25,7 @@ log = { version = "0.4.17", features = ["std"] }
 clap = { version = "4.1.4", features = ["derive", "env"] }
 regex = "1"
 ctrlc = { version = "3.0", features = ["termination"] }
+zip = "0.6.4"
```

To reduce the dependencies added by this crate:

```suggestion
zip = { version = "0.6.4", default-features = false, features = [ "aes-crypto" ] }
```

I have not tried if this still works with features set. If not, check https://github.com/zip-rs/zip/blob/master/Cargo.toml